src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  let sexPerson = 'wife';
+
+  if (sex === 'f') {
+    sexPerson = 'husband';
+  }
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        {age && (
+        <p className="Person__age">
+          I am
+          {' '}
+          {age}
+        </p>
+        )}
```

Comment:
please check the code style 
```suggestion
        {age && (
          <p className="Person__age">
            I am
            {' '}
            {age}
          </p>
        )}
```